.gitmodules

```diff
@@ -22,3 +22,6 @@
 	path = vendor/nanopb
 	url = https://github.com/nanopb/nanopb.git
 	ignore = untracked
+[submodule "vendor/artifacts"]
```

Comment:
I don't like this being included as a submodule. It makes impossible for outsiders to properly check-out the project. We might want to become this part of the internal Dockerfile. Let's discuss this IRL.